FASTER/Models/ServerCfg.cs

```diff
@@ -811,13 +764,94 @@ public string ServerCfgContent
                 RaisePropertyChanged("ServerCfgContent");
             }
         }
-
+        
         public ServerCfg()
         {
             if(string.IsNullOrWhiteSpace(serverCfgContent))
             { ServerCfgContent = ProcessFile(); }
         }
 
+        public object LogObjectNotFound { get; private set; }
+        public object SkipDescriptionParsing { get; private set; }
+        public object IgnoreMissionLoadErrors { get; private set; }
+        public object QueueSizeLogG { get; private set; }
+
+    [Serializable]
+    public class AdvancedOptions : INotifyPropertyChanged
+    {
+        private bool   logObjectNotFound       = true;       // logging enabled
+        private bool   skipDescriptionParsing  = false;      // Parse description.ext
+        private bool   ignoreMissionLoadErrors = false;      // Do not ingore errors
+        private int    queueSizeLogG           = 0;          // If a specific players message queue is larger than <Value> and '#monitor' is running, dump his messages to a logfile for analysis
+
+            public bool LogObjectNotFound
+        {
+            get => logObjectNotFound;
+            set
+            {
+                logObjectNotFound = value;
+                RaisePropertyChanged("LogObjectNotFound");
+            }
+        }
+
+            public bool SkipDescriptionParsing
+        {
+            get => skipDescriptionParsing;
+            set
+            {
+                skipDescriptionParsing = value;
+                RaisePropertyChanged("SkipDescriptionParsing");
+            }
+        }
+
+            public bool IgnoreMissionLoadErrors
+        {
+            get => ignoreMissionLoadErrors;
+            set
+            {
+                ignoreMissionLoadErrors = value;
+                RaisePropertyChanged("IgnoreMissionLoadErrors");
+            }
+        }
+
+            public int QueueSizeLogG
+        {
+            get => queueSizeLogG;
+            set
+            {
+                queueSizeLogG = value;
+                RaisePropertyChanged("QueueSizeLogG");
+            }
+        }
+		
+		private string advancedOptionsContent;    
+
+        public string AdvancedOptionsContent
+        {
+            get => AdvancedOptionsContent;
+            set
+            {
+                AdvancedOptionsContent = value;
+                RaisePropertyChanged("AdvancedOptionsContent");
+            }
+        }
+
+        public Advancedoptions()
```

Comment:
[L839-L851](https://github.com/Foxlider/FASTER/pull/186/files#diff-9babbf2c42ae3235cb955d5b6eb360c09935b7606dd78683185323b5a6888263R839-R851) needs to be changed